crates/holochain_zome_types/src/version.rs

```diff
@@ -0,0 +1,13 @@
+use crate::prelude::*;
+
+/// The version of the API so that wasm host/guest can stay aligned.
+///
+/// Something roughly along the lines of the pragma in solidity.
+///
+/// @todo implement this
+#[derive(Debug, Serialize, Deserialize)]
+pub enum Version {
```

Comment:
```suggestion
pub enum ZomeApiVersion {
```

Can we prefix it with something descriptive? There are several other "versions" coming soon, including versions of the DNA and App Manifest formats, as well as versions of DNAs themselves.

Reply:
sure